docs/usecases/bids-fmriprep-workflow-NP.sh

```diff
@@ -222,6 +222,7 @@ containers/scripts/freeze_versions --save-dataset=^ \
     poldracklab-ds003-example=0.0.3 \
     bids-mriqc=0.15.0 \
     bids-fmriprep=1.4.1
+( cd containers/ ; git clean -dfx ; )
```

Comment:
Note: this is a temporary workaround to remove .datalad/config-e created by above freeze_versions script. Proper solution sounds fix that script to stay compatible with osx.